src/jquery.selectric.js

```diff
@@ -776,6 +790,10 @@
     open: function(e) {
       var _this = this;
 
+      if ( _this.options.nativeOnMobile && _this.utils.isMobile()) {
+        return false;
```

Comment:
My first approach was to stop rendering `selectric-items` list because in case of `nativeOnMobile` there is no need for but I found this would result in bigger refactoring. So I added a simple quick fix here...